dags/solutions_team/configs/tensorflow/solutionsteam_tf_release_supported_config.py

```diff
@@ -289,19 +301,21 @@ def get_tf_dlrm_config(
       },
   }
 
-  test_name = f"tf_{MAJOR_VERSION}_{MINOR_VERSION}_dlrm_criteo"
-  benchmark_id = f"{test_name}-v{tpu_version.value}-{tpu_cores}"
-  # Add default_var to pass DAG check
-  # TODO(ranran): replace Variable.get() to XCOM when it applies
-  tpu_name = Variable.get(benchmark_id, default_var=None) if is_pod else "local"
-  env_variable = export_env_variable(is_pod, is_pjrt)
+  model_dir = "/tmp/"
+  if is_v5p:
+    params_override["trainer"]["pipeline_sparse_and_dense_execution"] = "true"
+    tpu_id = Variable.get(benchmark_id, default_var=None)
+    model_dir = gcs_bucket.BASE_OUTPUT_DIR + f"/dlrm/v5p/{benchmark_id}"
```

Comment:
Wondering why model_dir is on gcs for v5p?

Also, could you update it to something like 

```
gcs_bucket.BASE_OUTPUT_DIR + f"{Team.SOLUTIONS_TEAM.value}/dlrm/v5p/{benchmark_id}"
```

We actually added a new feature to use default gcs location. Could you take a quick look and see if that fits as well? One example is [here](https://github.com/GoogleCloudPlatform/ml-auto-solutions/blob/master/dags/solutions_team/configs/flax/solutionsteam_flax_latest_supported_config.py#L188): 
* check `gcs_subfolder` to pass in team's info
* check `metric_config.SshEnvVars.GCS_OUTPUT.value` to get this value
* check `use_runtime_generated_gcs_folder` as True

Reply:
Thanks for the explanation! Yeah, if it has multi-hosts, using gcs bucket for ckpts makes sense. 

Will adding `gcs_subfolder` and `metric_config.SshEnvVars.GCS_OUTPUT.value` works for you. No need to call `use_runtime_generated_gcs_folder` this is for metric processing.

Reply:
Yup, that works, will fix!

(I thought the `metric_config` was an instance when I read that first but it's actually the package name.)

Reply:
Discussed offline. We will use default location later as env is not imported if did not indicate `use_runtime_generated_gcs_folder` flag.